server/src/data-processors/user/get-user-processor.js

```diff
@@ -0,0 +1,26 @@
+const requiredDependency = require('../../global-helpers/required-dependency');
+const requiredParam = require('../../global-helpers/required-parameter');
+const mockResponseData = require('../../database/mock-response-data') // Will remove once dbQuery helper complete
+
+async function getUserProcessorFunction({
+  extractTokenFromAuthHeader = requiredDependency('extractTokenFromAuthHeader'),
+  verifyAuth = requiredDependency('verifyAuth'),
+  dbQuery,
+} = {},
```

Comment:
Why add an object as a default value for params when `extractTokenFromAuthHeader` and `verifyAuth` already have defaults in place?

Reply:
Makes sense! Thanks boi!